nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -605,8 +615,9 @@ in {
               'datadirectory' => '${cfg.home}/data',
               'skeletondirectory' => '${cfg.skeletonDirectory}',
               ${optionalString cfg.caching.apcu "'memcache.local' => '\\OC\\Memcache\\APCu',"}
-              'log_type' => 'syslog',
-              'log_level' => '${builtins.toString cfg.logLevel}',
+              'log_type' => '${cfg.logType}',
+              ${optionalString cfg.logFile "'logfile' => '${cfg.logFile}',"}
```

Comment:
This doesn't evaluate, in *any* case:

```
error: value is a string while a Boolean was expected
```

with `logFile` being declared and

```
error: value is null while a Boolean was expected
```

Please fix this accordingly and test the changes e.g. using a VM test (you can perform a single run via e.g. `nix-build nixos/tests/nextcloud -A basic23`).

Also, please make sure that your commit messages adhere to our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Reply:
The current change doesn't evaluate. As long as that's still the case, this cannot be merged.